### PR TITLE
Update desktop.py

### PR DESCRIPTION
**Pull request by PipKat**
Use single back tick  for classes, functions.
Minor edits.

### CODE REVIEW

File: pyaedt/desktop.py [hunk 1]
```diff
@@ -71,17 +54,16 @@
 
 
 def exception_to_desktop(self, ex_value, tb_data):
-    """Writes the trace stack to the desktop when a python error occurs.
+    """Writes the trace stack to the desktop when a Python error occurs.
     
-    It adds the message to the AEDT global message manager and to the log file (if present).
+    The  message is added to the AEDT global Message Manager and to the log file (if present).
```
Comment (akaszynski) [inline pyaedt/desktop.py:59]:
```suggestion
    The message is added to the AEDT global Message Manager and to the log file (if present).
```

File: pyaedt/desktop.py [hunk 2]
```diff
@@ -318,6 +301,9 @@ class Desktop:
 
     >>> desktop = Desktop("2021.1")
     >>> hfss = pyaedt.Hfss()
+    
+    The previous example initializes the `Desktop` module to the latest AEDT version 
+    installed on your machine in graphical mode and initializes HFSS.
```
Comment (akaszynski) [inline pyaedt/desktop.py:306]: This section seems unnecessary.  Also, it's not correct as we're requesting the version of AEDT to be `"2021.1"`.  Let's remove this section.

**Review by akaszynski — APPROVED** [review]
Approve with suggestions.

File: pyaedt/desktop.py [hunk 3]
```diff
@@ -318,6 +301,9 @@ class Desktop:
 
     >>> desktop = Desktop("2021.1")
     >>> hfss = pyaedt.Hfss()
+    
+    The previous example initializes the `Desktop` module to the latest AEDT version
```
Comment (PipKat) [inline pyaedt/desktop.py:305]:
```suggestion
```

File: pyaedt/desktop.py (hunk 2 above)
Comment (PipKat) [inline pyaedt/desktop.py:306]:
```suggestion
```

**Pull request merged**